src/main/java/de/tum/in/www1/artemis/repository/StudentParticipationRepository.java

```diff
@@ -1210,4 +1211,54 @@ SELECT COALESCE(AVG(p.presentationScore), 0)
                 AND p.presentationScore IS NOT NULL
             """)
     double getAvgPresentationScoreByCourseId(@Param("courseId") long courseId);
+
+    /**
+     * Get aggregated feedback details for a given exercise, including the count and relative count
+     * of each unique feedback detail text and test case name. The query considers only the latest
+     * result for each student participation when calculating the counts and relative counts, and
+     * excludes participation that are in practice mode (i.e., `testRun` is `TRUE`).
+     *
```

Comment:
This will be ignored in the preview otherwise
```suggestion
     * <br>
```